README.md

```diff
@@ -0,0 +1,41 @@
+# Cincinnati
+
+Cincinnati is a service to transform update information as modeled in a Docker registry (such as Quay.io) into a graph representation that can be consumed by update clients. It is the spiritual successor to the Omaha protocol used to udate Google Chrome and Google Chrome OS based operating systems.
+
+## Common Usage
+
+```sh
+# docker run -d -p 8080:8080 <image> --address 0.0.0.0 --registry https://quay.io --repository <namespace/reponame>
```

Comment:
Please drop the `docker run` invocations and just use `cargo run`.

Reply:
My image doesn't package cargo or the rust build system, just the binaries. I would be happy to just reference the binary names.